modules/spam/text.py

```diff
@@ -291,15 +291,13 @@ def brainfuck(self, update: Update, context: CallbackContext) -> None:
         max_cell_value = 255
         do_wrapping = True
 
-        if not len(context.args) or len(context.args) > 2:
+        if not len(context.args):
             update.message.reply_text("Usage: /brainfuck code [input]")
             return
-        elif len(context.args) == 1:
-            instr = context.args[0]
-            inputs = None
-        else:  # == 2
+        else:
             instr = context.args[0]
-            inputs = list(context.args[1])
+            # assumes that all words of input were separated by exactly one space
+            inputs = list(' '.join(context.args[1:]))

```

Comment:
Why?

Reply:
Well, i want to be able to include spaces in the input and this is the only way I could come up with. It's not great cause it will reduce any amount of whitespace to one space but I couldn't find a way to get the original full text of the message

Reply:
Oh, you can `_, instr, inputs = update.message.split(" ", 2)`